modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} installs certain CSI drivers by default, giving users storage options that are not possible with in-tree volume plug-ins.
```

Comment:
Do we need to spell out the first instance of CSI here for clarity. There doesn't seem to be a consolidated viewpoint on doing this in OCP, but if a user reaches this doc from Google will they know what that acronym stands for? I always spell out the first instance of an acronym in all of my topics, even if they are nested together in the same assembly, and then proceed with acronym throughout for this reason. But it's also a hard ingrained habit from past roles too. It's fine as it, just some thoughts :)

Reply:
Yes, I'm in total agreement. I try to spell out on first instance at the module level but missed this one, now fixed. Thanks!